src/drawerbase.js

```diff
@@ -113,10 +101,8 @@ $.DrawerBase = class DrawerBase{
         this._checkForAPIOverrides();
     }
 
+    // protect the canvas member with a getter
```

Comment:
This is just so we can't set the canvas? Seems like we have lots of properties we don't want people to set... Why single this one out for extra protection? I'm not against it... Just curious.

Reply:
Thinking about the possibility that plugin drawers might be how new rendering behavior can be created in the future, I thought this might enforce the API a bit more strictly. It probably doesn't really matter...

Reply:
Cool. Either way is fine with me.